doc/how_to_guides/templates.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Templates
+description:
+type: how_to_guide
+created_at: 2024-11-21
+---
+
+# Templates
+
+## Overview
+As you should know from our documentation of their [usage](/usage.md),
+creating a Onesie already involves generating a basic template. This
+section will discuss the more specifically customized templates we
+create for easy re-use when we expect the core functions of a Onesie
+to be repeated in the future.
+
+## Usage
+
+### Finding Templates To Use
+Writing a Onesie for a common use-case? Check to see if a template
+already exists in
+[onesie/templates](https://github.com/watermelonexpress/benchprep-v2/tree/release_candidate/onesie/templates). If not, consider creating
```

Comment:
This is very specific to a consumer of Onesie, which if this document lived in `benchprep-v2`, would make sense.

Can we update this to be more generalized? The path to the templates directory in this gem is hardcoded, so we can refer folks to look in a consumers directory listed here: https://github.com/watermelonexpress/onesie/blob/29a19b95c79e503cc5765b2b4dc89d0d5abcc940/lib/generators/onesie/task_generator.rb#L34